src/test/app/Escrow_test.cpp

```diff
@@ -671,18 +674,84 @@ struct Escrow_test : public beast::unit_test::suite
     }
 
     void
-    testMeta()
+    testMetaAndOwnership()
     {
-        testcase ("Metadata");
-
         using namespace jtx;
         using namespace std::chrono;
-        Env env(*this, with_features(featureEscrow));
 
-        env.fund(XRP(5000), "alice", "bob", "carol");
-        env(condpay("alice", "carol", XRP(1000), makeSlice(cb1), env.now() + 1s));
-        auto const m = env.meta();
-        BEAST_EXPECT((*m)[sfTransactionResult] == tesSUCCESS);
+        auto const alice = Account("alice");
+        auto const carol = Account("carol");
+
+        {
+            testcase ("Metadata & Ownership (without fix1523)");
+            Env env(*this, with_features(featureEscrow));
```

Comment:
I'd lead toward `all_features_except()` in this case and then just the default `Env` (with all features) for the remaining tests, but I realize there is some refactoring by @scottschurr that might impact some of this too.

Reply:
I squinted at the features handling in these unit tests.  I figured I'd make a tidying pass on this test after my other changes are merged.  Which is not to say I disagree with @mellery451's  suggestions.  They are good ones...

Reply:
Just to be clear, are you suggesting that this be: `Env env(*this, all_features_except(fix1523))`?

Reply:
@nbougalis yes, that's what I was thinking